pytorch_lattice/layers/rtl.py

```diff
@@ -0,0 +1,274 @@
+"""Random Tiny Lattice module for use in calibrated modeling.
+
+PyTorch implementation of a RTL layer.
+This layer takes several inputs which would otherwise be slow to run on a single lattice
+and runs random subsets on an assortment of Random Tiny Lattices as an optimization.
+"""
+import logging
+from typing import List, Optional
+
+import numpy as np
+import torch
+
+from ..enums import Interpolation, LatticeInit, Monotonicity
+from .lattice import Lattice
+
+
+class RTL(torch.nn.Module):
+    """An RTL Module.
+
+    Layer takes a number of features that would otherwise be too many to assign to
+    a single lattice, and instead assigns small random subsets of the features to an
+    ensemble of smaller lattices. The features are shuffled and uniformly repeated
+    if there are more slots in the RTL than features.
+
+    Attributes:
+      - All `__init__` arguments.
+      - _lattice_layers: `dict` of form `{monotonic_count: (lattice, groups)}` which
```

Comment:
Do not include private attributes in the attributes list in the docstring. Please remove this.